src/meta_ext.rs

```diff
@@ -229,6 +232,17 @@ impl<'a> Pallet<'a> for PalletMeta {
         let name = self.name.decoded();
         name
     }
+
+    fn get_calls(&self) -> Option<&PalletCallMetadata<PortableForm>> {
+        self.calls.iter().find(|_call| {
+            true
+        })
+    }
+
+    fn ty_id(&self) -> u32 {
+        dbg!(self);
+        0u32
```

Comment:
which type id would be suitable to return here?

Reply:
None. Pallets don't have `ty_id`s, but `index`es. That was a method written early in the process, a naive attempt to _"find"_ the type ID for a pallet, and kept that function after discarding its usage.

Removed that method from the trait.